service/pom.xml

```diff
@@ -54,11 +54,21 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>de.starwit</groupId>
+            <artifactId>vision-lib</artifactId>
+            <version>0.3.0</version>
```

Comment:
die dependency sollte noch in die parent pom...

Reply:
Done